configure.ac

```diff
@@ -2212,6 +2217,7 @@ SURICATA_BUILD_CONF="Suricata Configuration:
   Rust support (experimental):             ${enable_rust}
   Experimental Rust parsers:               ${enable_rust_experimental}
   Rust strict mode:                        ${enable_rust_strict}
+  Rust strict debug:                       ${enable_rust_debug}
```

Comment:
Is this still strict?

Reply:
No, mistake on my end. Should read 'Rust debug mode'